core/src/com/unciv/logic/map/TileMap.kt

```diff
@@ -129,6 +127,30 @@ class TileMap {
                     }
                 }.filterNotNull()
 
+    private fun getIfTileExistsOrNull(x: Int, y: Int) : TileInfo? {
+        if (contains(x, y)) {
+            return get(x, y)
+        }
+
+        if (mapParameters.worldWrap) {
```

Comment:
If(worldWrap) return null
...


To reduce indentation and increase readability

Reply:
done